CHANGELOG.md

```diff
@@ -36,6 +36,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Crash on `clear` when scrolled up in history
 - Entire screen getting underlined/stroke out when running `clear`
 
+### Removed
+
+- Support for non-utf8 C1 escapes 
```

Comment:
`non-utf8` -> `non-UTF-8`?

Our other changelog entries mentioning UTF-8 do so in the proper case ;-)

If that looks as strange to you as it does to me, maybe change the entire message to `- Support for 8-bit C1 escapes`? (I think 8-bit is the proper name after briefly skimming the formerly-mentioned issue)

Reply:
technically speaking utf8 and UTF-8 are both fine and in use, so the current changelog message isn't controversial in my opinion and reflects the naming for similar things like mime types (e.g. `text/plain;charset=utf-8`).

Reply:
I'm fine either way, but I figured consistency is key.

Reply:
Yeah I chose the lowercase naming because `non-UTF-8` looks strange.

In theory `8-bit` is accurate too, but we've never had 8-bit UTF-8 escapes, so I wanted to choose the word that's more specific here.